src/components/widgets/Number.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import Header from './Header'
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+class Number extends Component {
+  render() {
+    const { title, x, y, statistic, goal } = this.props
+    const positionStyle = { top: x, left: y }
+
+    const progressRatio = ( statistic / goal ) * 100
```

Comment:
This still includes entirely too much calculation here; that's why I suggested the separate methods in my previous review.  When you list the responsibilities of the `render` method, I would suggest they are: calculate progress (what's a progress ratio?), coerce that calculation into a string, and render.  If you were naming this method by what it did, what would it's name be? Probably something like `calculateProgressAndCoerceItToAStringAndRender`. 

Please refactor to appropriately separate the responsibilities - `render` should only be dealing with rendering, put the calculation logic elsewhere.  See earlier review messages for an idea.